crates/graphql/src/schema/objects/store_creator.rs

```diff
@@ -18,25 +19,53 @@ impl StoreCreator {
         &self.creator_address
     }
 
+    pub fn twitter_handle(&self) -> Option<&str> {
+        self.twitter_handle.as_deref()
+    }
+
     pub async fn preview(&self, context: &AppContext) -> FieldResult<Vec<Nft>> {
         context
             .collection_loader
             .load(self.creator_address.clone().into())
             .await
             .map_err(Into::into)
     }
+
+    pub async fn profile(&self, ctx: &AppContext) -> FieldResult<Option<TwitterProfile>> {
+        let twitter_handle = match self.twitter_handle {
+            Some(ref t) => t.clone(),
+            None => return Ok(None),
+        };
+
+        ctx.twitter_profile_loader
+            .load(twitter_handle)
+            .await
+            .map_err(Into::into)
+    }
+
+    pub async fn stats(&self, context: &AppContext) -> FieldResult<Option<StoreCreatorStats>> {
+        context
+            .collection_stats_loader
+            .load(self.creator_address.clone().into())
+            .await
+            .map_err(Into::into)
+    }
```

Comment:
stats isn't quite right. We want counts for the collection previews which your query is doing but not a sales state.
```suggestion
    pub async fn nft_count(&self, context: &AppContext) -> FieldResult<Option<i32>> {
        context
            .collection_count_loader
            .load(self.creator_address.clone().into())
            .await
            .map_err(Into::into)
    }
```

Reply:
@kespinola Here If we are converting FieldResult type from Option<StoreCreatorStats> to Option<i32>, then do we require the dataloader anymore? Should not we also remove the dataloader in this case?